autobotAI_integrations/integrations/google_chat/webhook_client.py

```
@@ -83,16 +72,11 @@ def send(self, title, body, buttons=[]):
-            resp = request.urlopen(req, timeout=10)
-            return {
-                "success": True,
-                "status_code": resp.getcode(),
-                "data": json.loads(resp.read().decode("utf-8"))
-            }
-        except request.HTTPError as e:
-            error_body = e.read().decode("utf-8")
-            raise Exception(f"Google Chat HTTP {e.code}: {e.reason}. Details: {error_body}")
+            resp = request.urlopen(req)
+            return resp
         except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)